FUNCTIONS.md

```diff
@@ -166,35 +166,135 @@ Transformation functions modify each series in a given serieslist independently.
 This function computes a numerical derivative for a given timeseries. Each value will be updates to be `y[i] = (x[i] - x[i-1]) / interval` where `interval` is the time between samples, measured in seconds.
 The first value is assigned 0.
 
+Consider performing the query `transform.derivative( disk_usage )` with a resolution of 10 seconds (30 seconds is the default resolution), where `disk_usage` is defined like this:
+
+| (series†) | metric name | tags        | series values |
+|:---------:|:-----------:|:-----------:|:-------------:|
+| (A1)      | disk_usage  | app: ui     | 300  310  315 |
+| (A2)      | disk_usage  | app: server | 440  435  450 |
+
+The query `transform.deriative( disk_usage )` results in:
+
+| (series†) | metric name                      | tags        | series values |
+|:---------:|:--------------------------------:|:-----------:|:-------------:|
+| (A1)      | transform.derivative(disk_usage) | app: ui     | 0   1    0.5  |
+| (A2)      | transform.derivative(disk_usage) | app: server | 0  -0.5  1.5  |
+
 ##### `transform.integral(list)`
 
-This function computes a numerical integral for a given timeseries. Each value will be the sum of the values up to it (including itself), a left Riemann integral.
+This function computes a numerical integral for a given timeseries. Each value will be the sum of the values up to it (including itself).
 This quantity will be scaled, interpreting each value in the series as having units of `events / second`, and producing something with the units of `events`.
 If you do not want this behavior, use `transform.cumulative` instead, which does not have scaling behavior.
 
 `NaN` values are treated as 0.
 
+Consider the query `transform.integral( request_rate )` with a resolution of 10 seconds (30 seconds is the default resolution) on the metric `request_rate` detailed below:
+
+| (series†) | metric name  | tags        | series values |
+|:---------:|:------------:|:-----------:|:-------------:|
+| (A1)      | request_rate | app: ui     | 30  25  20    |
```

Comment:
can we use backticks for the `series values` column? I think it makes them a bit easier to read.